docker-compose.yml

```diff
@@ -0,0 +1,311 @@
+services:
+  credentials:
+    container_name: credentials
+    command:
+      - "/bin/sh"
+      - "/scripts/make_credentials.sh"
+    depends_on:
+      rabbitmq:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+    environment:
+      - PGPASSWORD=rootpass
+    image: ubuntu:kinetic
+    networks:
+      - secure
+    volumes:
+      - ./scripts:/scripts
+      - shared:/shared
+
+  data_loader:
+    container_name: data_loader
+    command:
+      - "/bin/sh"
+      - "/load_data.sh"
```

Comment:
Is this actually correct, or should it be `/bin/sh /load_data.sh`?

Reply:
It doesn't matter here, command and arg is separated for visibility reasons